internal/hosts/metadata.go

```diff
@@ -2,11 +2,11 @@ package hosts
 
 type Metadata struct {
 	Cluster         string `mapstructure:"ha-cluster,omitempty"`
-	ClusterId       string `mapstructure:"ha-cluster-id,omitempty"`
+	ClusterID       string `mapstructure:"ha-cluster-id,omitempty" json:"ClusterId"`
```

Comment:
Where do we use this `Metadata` struct?
I would bet that nowhere hehe

Reply:
Yes, yes, I was not complaining about the change. Just saying that this struct is most probably some legacy code we had when we used consul, but not it is not used anywhere, so removing it (together with the test) should be safe